homeassistant/components/tado/const.py

```diff
@@ -139,20 +143,36 @@
     HVACMode.FAN_ONLY: CONST_MODE_FAN,
 }
 
+HA_TO_TADO_FAN_MODE_MAP_LEGACY = {
+    FAN_AUTO: CONST_FAN_AUTO,
+    FAN_OFF: CONST_FAN_OFF,
+    FAN_LOW: CONST_FAN_LOW_LEGACY,
+    FAN_MEDIUM: CONST_FAN_MIDDLE_LEGACY,
+    FAN_HIGH: CONST_FAN_HIGH_LEGACY,
+}
+
 HA_TO_TADO_FAN_MODE_MAP = {
     FAN_AUTO: CONST_FAN_AUTO,
     FAN_OFF: CONST_FAN_OFF,
-    FAN_LOW: CONST_FAN_LOW,
-    FAN_MEDIUM: CONST_FAN_MIDDLE,
-    FAN_HIGH: CONST_FAN_HIGH,
+    FAN_LOW: CONST_FAN_LEVEL1,
```

Comment:
Please use proper formatting. For instance: `CONST_FAN_LEVEL_1`.

Reply:
Yes, it's done. My bad.